multibuildingdetector/__main__.py

```diff
@@ -19,7 +19,7 @@ def load_config(path):
         return yaml.load(f)
 
 
-def run(input, output, batch_size, iterator='SerialIterator', device=-1,
+def run(input, output, batch_size, train_split, iterator='SerialIterator', device=-1,
```

Comment:
Please add a default value eg. 80/20 for the split and this line is too long >79.